ingest/src/main/java/com/connexta/ingest/client/StoreClient.java

```diff
@@ -71,8 +70,8 @@ public String getFilename() {
     final URI location;
     try {
       location = restTemplate.postForLocation(storeEndpoint, request);
-    } catch (final RestClientException e) {
-      throw new StoreException("Unable to POST to store endpoint", e);
+    } catch (Exception e) {
+      throw new StoreException("Store exception", e);
```

Comment:
✏️ Move this string to the constructor of StoreException (same for TransformException).
```java
public StoreException(Throwable cause) {
  super("Unable to store", cause);
}
```
```java
public TransformException(Throwable cause) {
  super("Unable to transform", cause);
}
```